lotus/sem_ops/sem_schema.py

```diff
@@ -0,0 +1,109 @@
+from typing import Callable
+
+import pandas as pd
+
+import lotus
+from lotus.models import LM
+from lotus.templates import task_instructions
+from lotus.types import LMOutput, SemanticSchemaOutput, SemanticSchemaPostprocessOutput
+
+from .postprocessors import schema_postprocess
+
+
+def sem_schema(
+    docs: list[str],
+    model: LM,
+    columns: list[str],
+    col_description: list[str],
```

Comment:
We should remove `col_description` everywhere since its not used in the prompt formatter. Generally we can remove this because the LM should be good enough to infer the meaning of columns.